### PR TITLE
Use more idiomatic throttling code

### PR DESCRIPTION
**Pull request by mourner**
Made the throttling code simpler and more idiomatic while investigating #5185.

 - [x] briefly describe the changes in this PR
 - [ ] write tests for all new functionality
 - [x] manually test the debug page

### CODE REVIEW

File: src/util/util.js [hunk 1]
```diff
@@ -222,6 +222,32 @@ exports.bindAll = function(fns: Array<string>, context: Object): void {
 };
 
 /**
+ * Throttle the fiven function to run at most every `time` milliseconds.
```
Comment (ChrisLoer) [inline src/util/util.js]: nit: fiven -> given

**Comment by mourner** [comment]
@ChrisLoer sure — if #5150 is close to shipping, feel free to close the PR.

**Comment by mourner** [comment]
Not sure what's going on with the freezing tests though, I thought this was something on the Circle side...

**Comment by mourner** [comment]
Closing since this will be irrelevant with #5150. We can revisit when we need throttling code again.

**Pull request closed without merging**